pom.xml

```
@@ -22,11 +22,11 @@
 
 		<jackson.version>2.19.2</jackson.version>
 		<cxf.version>4.1.3</cxf.version>
-		<spring.version>6.2.11</spring.version>
-		<spring-security.version>6.4.6</spring-security.version>
+		<spring.version>6.2.12</spring.version>
+		<spring-security.version>6.4.12</spring-security.version>
 		<spring-integration.version>6.5.1</spring-integration.version>
 		<spring.boot.version>3.5.5</spring.boot.version>
```

```suggestion
		<spring-integration.version>6.5.3</spring-integration.version>
		<spring.boot.version>3.5.7</spring.boot.version>
```

Die kunnen dan ook wel mee toch?